styles.css

```diff
@@ -83,3 +104,149 @@ ul.socials {
   justify-content: space-between;
   max-width: 170px;
 }
+
+ul.works_grid {
+  display: grid;
+  grid-template-columns: 1fr;
+  padding: 114px 24px;
+  grid-row-gap: 88px;
+  list-style-type: none;
+}
+
+ul.works_grid ul {
+  list-style-type: none;
+  padding-left: 0;
+}
+
+ul.works_header {
+  display: flex;
+  padding-left: 0;
+  align-items: center;
+}
+
+ul.works_stack li {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 4px 12px;
+  background: #ebebff;
+  border-radius: 8px;
+  margin-right: 8px;
+}
+
+ul.works_header li {
+  text-transform: capitalize;
+  font-size: 13px;
+}
+
+ul.skills_dropdown li {
+  display: flex;
+  background: #f7f7f9;
+  border-radius: 8px;
+  margin-bottom: 12px;
+  padding: 9px 12px;
+  align-items: center;
+}
+
+ul.works_header li:first-child {
+  text-transform: uppercase;
+  font-weight: 600;
+  line-height: 16px;
+  color: #344563;
+}
+
+ul.works_header span {
+  width: 8px;
+  height: 8px;
+  background: #c1c7d0;
+  margin: 0 12px;
+  border-radius: 50%;
+}
+
+li.works_card {
+  display: flex;
+  flex-direction: column;
+  padding: 16px;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  align-items: center;
+}
+
+div.works_details h3 {
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172b4d;
+}
+
+ul.works_stack {
+  display: flex;
+  margin-bottom: 12px;
+}
+
+button.works_btn,
+.about_btn {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 12px;
+  border: 1px solid #6070ff;
+  border-radius: 8px;
+  background: #fff;
+  font-weight: 500;
+  font-size: 17px;
+  text-align: center;
+  letter-spacing: 0.03em;
+  color: #396df2;
+  margin: 24px 0 28px;
+}
+
+button.works_btn:hover,
+button.about_btn:hover {
+  background: #6070ff;
+  box-shadow: 0 8px 16px rgba(64, 83, 252, 0.24);
+  color: #fff;
+}
+
+#about {
+  display: flex;
+  flex-direction: column;
+  padding: 114px 24px;
+}
```

Comment:
- [ ] Kindly give your about section a `border` and a `border-top-right-radius` as seen in your chosen Figma design 🙏 

Figma Design | Your Design
------------- | -------------
![image](https://user-images.githubusercontent.com/74814780/206481576-2e03dc02-c8c9-4526-8e44-30317609ffdc.png)  |  ![image](https://user-images.githubusercontent.com/74814780/206481358-047bf8ce-9b9b-4c78-84ee-ee5e8dc6da7a.png)

Reply:
Thanks @Ugboaja-Uchechi. Please as regards this, I noticed there was no border property assigned to the about section from the Figma file. And also for the border-top-right-radius, I only assumed a value of 100px as I can't seem to find it on the Figma file

Reply:
Hi @AyoMoses1, not all style properties are written in the Figma design that's why I stated the style properties for you so you can get a better idea. I hope this clears it up for you 😊